src/main/java/com/ecore/roles/client/model/Team.java

```diff
@@ -0,0 +1,35 @@
+package com.ecore.roles.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Id;
+import java.util.List;
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+public class Team {
+
+    @Id
+    @JsonProperty
```

Comment:
Add all the JSON properties names in the @JsonProperty annotations, in order to don't get a code refactoring in the fields names leading to a mismatch in the API clients.